src/StartScreen.tsx

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { StyleSheet, Text, View, Image, TouchableHighlight } from 'react-native';
+
+export const StartScreen = () => {
+    const goRead = () => {}
+
+    return (
+        <View style={styles.startScreen}>
+            <Image
+                style={styles.logo}
+                source={require('../assets/wordmark.png')}
+            />
+            <TouchableHighlight
+                onPress={goRead}
```

Comment:
Ok, so i guess in the future we'll make clicking the button call navigate() to a Book Read screen or something?

This is fine for now.

Reply:
👍 exactly